meilisearch-http/Cargo.toml

```diff
@@ -28,6 +28,7 @@ crossbeam-channel = "0.4.2"
 env_logger = "0.7.1"
 flate2 = "1.0.16"
 futures = "0.3.4"
+heed = "0.8.0"
```

Comment:
Sorry but I need:
```
use heed::RoTxn;
use heed::RwTxn;
```

Reply:
but it's a good point, the only way to avoid this is to move backup in core engine

Reply:
wait, why does this belong to http in the first place? Backup is independent from the core's front end, no?

Reply:
It could be independent but it would need to duplicate code principally to read database or make a big refacto...
Meilisearch http already use transactions everywhere, I could avoid importation but I would have to write everything in the same function..

Reply:
then I suggest you use these instead: https://github.com/meilisearch/MeiliSearch/blob/master/meilisearch-core/src/database.rs#L24-L28
they are reexported at the root of the crate

Reply:
@MarinPostma done 😁